### PR TITLE
Add simple grammar and code corrections

### PR DESCRIPTION
**Pull request by rafaeldolfe**
First of all thanks for you beautiful implementation of parallelization of A* in Unity. I am currently trying to implement it in my game and so far it's going smoothly! This pull request is just for touching up on the README.md a bit. I got an error when I copied your code, so I fixed that. But then I noticed some grammar errors too so I thought I should fix those too.

I have currently implemented a feature into your code. I need the path calculated by the thread to be returned to me as a List<PNode>, and not to execute a void function (although I might still use that for debugging). So I implemented a PathfindingCoroutine which waits on the thread to finish, and then returns the final path to whichever function requested the pathfinding algorithm. All previous functionality is preserved. Let me know if that sounds like an interesting feature, in that case I could touch it up and make a pull request of it.

Cheers and thanks again,
Rafael

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,4 +158,4 @@ This uses [BlueRaja's](https://github.com/BlueRaja/High-Speed-Priority-Queue-for
 I can't guarantee that this is bug-free, or that it will necessarily work for you. I've done my best to test and optimize this, but it is far from perfect. If you find any bugs, report them here on Github by opening an issue.
 
 ## Contribution
-Any contiribution is welcome. If you make any changes that you think other people would benefit from, create a Pull Request and I'll merge it as soon as possible. 
+Any contributions is welcome. If you make any changes that you think other people would benefit from, create a Pull Request and I'll merge it as soon as possible.
```
Comment (Epicguru) [inline README.md:161]: I think that was already correct. It means:
_Any (single) contribution is welcome._

If it were to be changed, it should probably be changed to:
_Any contributions **are** welcome._

**Review by Epicguru — APPROVED** [review]

**Comment by Epicguru** [comment]
Thanks for the PR. I don't really agree with the last grammar change, but I'll merge it anyway.

I'm interested in taking a look at the co-routine feature. If it works reliably, I'll merge that too.

As you may have noticed, I haven't worked on this project in 2 years. I'm currently working on a complete re-write, for a new game, using all the things I learned from this project to increase performance and ease-of-use. Once it's complete, I'll release it for free.

**Comment by rafaeldolfe** [comment]
Wow sorry about the last grammar change, you are right lol. Dunno why I added an s there.

Tbh since you do value performance a lot (which is good) I wonder if my coroutine solution will be of any value. I'll make a pull request if the final version of it seems very solid.

**Comment by Epicguru** [comment]
Ok, thanks. I'll close this PR now then.

**Pull request merged**